controller/env/appenv.go

```diff
@@ -393,6 +393,12 @@ func (ae *AppEnv) WrapMiddleware(f AppMiddleware) mux.MiddlewareFunc {
 	}
 }
 
+func (ae *AppEnv) HandleGet(path string, f AppHandler, prs ...permissions.Resolver) {
+	handler := ae.WrapHandler(f, prs...)
+	ae.RootRouter.HandleFunc(path, handler).Methods(http.MethodGet)
```

Comment:
how is this not a utility function yet? :)

Reply:
Also - going forward how do we keep the openapi spec up to date? @andrewpmartinez will the spec generate the server code for us? so that we'd update the spec/server code first?

Reply:
Just in time for it be made irrelevant by the swagger work :) (maybe, I haven't dug into it much yet).

Reply:
When I get my worked merged the workflow will be:

1) write swagger
2) generate server code
3) wire business logic in handlers to REST API presentation.

Right now it is getting a little stale, but the goal is to get the new workflow out this week.